### ISSUE TITLE
Ability to silently install *.dmg files?

### ISSUE THREAD

**Issue by CartBlanche** [jonathanpeppers/boots#127]
Hi Jonathan, 
 Hope you and the family are well. 
Currently the Github CI doens't have the VS for Mac 2022 IDE installed on any of its instances/VMs. I was hoping to use `boots` temporarily until they update the images. Currently, the download links for VS for Mac 2022 link to *.dmg files - like https://visualstudio.microsoft.com/thank-you-downloading-visual-studio-22-for-mac/?sku=communitymac&ch=pre&rel=17. Can  `boots` silent install *.dmgs ?

**Comment by jonathanpeppers** [comment]
No, we just have `.pkg` support on Mac. If it's just a couple commands, might be easy to add to `boots`?

If you send a PR, I would be happy to push an update out. Thanks!

**Comment by CartBlanche** [comment]
Looks like the steps are:

```cd ~/Desktop

curl -O http://darwinports.opendarwin.org/downloads/DarwinPorts-1.2-10.4.dmg

hdiutil attach DarwinPorts-1.2-10.4.dmg

cd /Volumes/DarwinPorts-1.2/

sudo installer -pkg DarwinPorts-1.2.pkg -target "/"

hdiutil detach /Volumes/DarwinPorts-1.2/
```
I try it locally tomorrow to see if it works. If it does I'll send you a PR.
